prolix/maintenance-bundle/2.1/config/packages/lexik_maintenance.yaml

```diff
@@ -0,0 +1,4 @@
+lexik_maintenance:
+    driver:
+        class: '\Lexik\Bundle\MaintenanceBundle\Drivers\FileDriver'
+        options: {file_path: '%kernel.project_dir%/../var/cache/lock'}
```

Comment:
```suggestion
        options:
            file_path: '%kernel.project_dir%/../var/cache/lock'
```

Reply:
What about using: '%kernel.cache_dir%/lock'

Reply:
> What about using: '%kernel.cache_dir%/lock' ?

yes, you´re totally right. thx for the suggestion...